src/components/GroupHeader/GroupHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import cx from 'classnames';
+import { ChevronRight } from '../../assets/icons';
+import { useTheme } from '../../context';
+
+interface Props {
+  href: string;
+  text: string;
+}
+
+export function GroupHeader({ href, text }: Props) {
+  const theme = useTheme();
+
+  return (
+    <a
+      className={cx('flex flex-row items-center gap-[4px] hover:gap-[8px] transition-gap duration-300 ease-in-out', {
```

Comment:
```suggestion
      className={cx('flex flex-row items-center gap-1 hover:gap-2 transition-gap duration-300 ease-in-out', {
```

gap-1 => 4px
gap-2 => 8px
로 들어갑미다요